app/controllers/blacklight/oembed/embed_controller.rb

```diff
@@ -2,18 +2,28 @@ module Blacklight::Oembed
   class EmbedController < ActionController::Base
 
     def show
-      render json: { html: get_embed_content(params[:url]) }
+      render json: { html: get_embed_content(params[:url], additional_params) }
     end
 
     private
 
-    def get_embed_content url
+    def get_embed_content(url, add_params)
       begin
-        OEmbed::Providers.get(url).html.html_safe
+        OEmbed::Providers.get(url, **add_params).html.html_safe
       rescue OEmbed::NotFound
         response.status = 400
         ""
       end
     end
+
+    ##
+    # Allows for blacklight-oembed users to pass through additional configured
+    # parameters
+    def additional_params
+      params
+        .slice(*Blacklight::Oembed::Engine.config.additional_params)
```

Comment:
So, if I understand this correctly, any parameter keys are allowed in the `additional_params` which is why you're doing `slice.to_unsafe_h`? Does it make sense to restrict which keys are allowed in `config.additional_params`, or is it open-ended?

Reply:
It's open ended here for a consumer to whitelist these parameters as needed.

I can't really see why or what we would restrict to here at the gem level. Open to suggestions though.